SportsHub/SportsHub.DAL/Data/Configurations/UserEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SportsHub.Domain.Models;
+
+namespace SportsHub.DAL.Data.Configurations
+{
+    public class UserEntityTypeConfiguration : IEntityTypeConfiguration<User>
+    {
+        public void Configure(EntityTypeBuilder<User> user)
+        {
+            user.Property(x => x.Email)
+                .IsRequired(true)
+                .HasMaxLength(50)
+                .IsUnicode(false);
+
+            user.Property(x => x.Username)
+                .IsRequired(true)
+                .HasMaxLength(50)
+                .IsUnicode(false);
+
+            user.Property(x => x.DisplayName)
+                .IsRequired(true)
+                .HasMaxLength(50)
+                .IsUnicode(false);
+
+            user.Property(x => x.Pseudonym)
+                .IsRequired(false)
```

Comment:
as I remember we agreed to use just one Pseudonym or DisplayName

Reply:
Pseudonym is now removed.